v2/datastream-to-spanner/src/test/resources/SeparateShadowTableDatabaseIT/mysql-schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `alltypes` (
```

Comment:
we can use the same schema for single db tests and shadow table test to see that there is no difference

Reply:
+1
We should test separate Shadow table code with all the test cases which we have for currently, with exact same data.
Only the @Before method would change, but we would still need to copy the test files to run them in separate Shadow table mode.

These test cases are also needed to cover all datatypes being used as primary key.